src/SDK/Metrics/Stream/DeltaStorage.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Metrics\Stream;
+
+use function assert;
+use GMP;
```

Comment:
It's optional, the code works for up to 63 readers without `GMP` and returns no-op readers otherwise / most applications should use way less. Will add a composer suggestion (/ forgot it while porting it over).